simplesamlphp/config/authsources.php

```diff
@@ -0,0 +1,339 @@
+<?php
+
+$config = [
+
+    // This is a authentication source which handles admin authentication.
+    'admin' => [
+        // The default is to use core:AdminPassword, but it can be replaced with
+        // any authentication source.
+
+        'core:AdminPassword',
+    ],
+
+
+    // An authentication source which can authenticate against both SAML 2.0
+    // and Shibboleth 1.3 IdPs.
+    'cancergov-sp' => [
+        'saml:SP',
+
+        // The entity ID of this SP.
+        // Can be NULL/unset, in which case an entity ID is generated based on the metadata URL.
+        'entityID' => 'https://www-int-ac.cancer.gov/simplesaml/module.php/saml/sp/metadata.php/cancergov-sp',
+
+        // The entity ID of the IdP this SP should contact.
+        // Can be NULL/unset, in which case the user will be shown a list of available IdPs.
+        'idp' => 'https://authdev.nih.gov/SAML2/IDP',
+
+        // The URL to the discovery service.
+        // Can be NULL/unset, in which case a builtin discovery service will be used.
+        'discoURL' => null,
+
+        // The private key and certificate for signing the SAML XML data.
+        'privatekey' => 'saml.pem',
+        'certificate' => 'saml.crt',
+        'signature.algorithm' => 'http://www.w3.org/2001/04/xmldsig-more#rsa-sha256',
+
+        /*
```

Comment:
can you please remove the example cruft? We can get it from the blt code if needed.

Reply:
Done!